config/routes.rb

```diff
@@ -2,12 +2,10 @@
   
   post '/login' => 'accounts#login'
   post '/signup' => 'accounts#create'
-  get '/letters' => 'letters#index'
-  get '/letters/:id' => 'letters#show'
-  post '/letters' => 'letters#create'
-  get '/responses/:id' => 'responses#letter'
-  get '/response/:id' => 'responses#show'
-  post '/responses' => 'responses#create'
-  #hi from brian
+
+  resources :letters, only: [:index, :create, :show] do
+     resources :responses, only: [:index, :create, :show]
+  end 
+
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
-end
+end
```

Comment:
Make sure you have your linter and formatter running, there should always be a new line at the end of the file (this ensure it will run on more machines if we potentially push this to a server.

Reply:
Adding the line myself since I want to merge this in!